app/config/sources.inc.php

```diff
@@ -496,6 +496,11 @@
         'priority'          => 1,
         'supported_locales' => ['hi-IN'],
     ],
+    'mozorg/newsletter/opt-out-confirmation.lang' => [
```

Comment:
Is this the name of the file in the PR?
https://github.com/mozilla-l10n/www.mozilla.org/pull/234/files